TypeCobol/Compiler/Preprocessor/PreprocessorStep.cs

```diff
@@ -475,6 +475,14 @@ internal static IList<DocumentChange<IProcessedTokensLine>> ProcessTokensLinesCh
 
                 TUVienna.CS_CUP.Runtime.Symbol ppSymbol = directivesParser.parse();
 
+#if EUROINFO_RULES
+                if (document.CompilerOptions.ReportUsedCopyNamesPath != null && 
+                    document.CompilerOptions.ExecToStep <= ExecutionStep.Preprocessor) 
+                {
+                    continue;
+                }
+#endif
```

Comment:
I guess this is for performance reasons but this is ambiguous. When the option is set, we will end up with an incomplete PreprocessorStep and results in `CompilationDocument` will be partial.

Reply:
Yes it's reaaly for performance reasons specific to this option.

Reply:
So you're ok with the fact that the resulting `CompilationDocument` instance will end up incomplete when using this option ?

Reply:
Yes, but another condition is that ExecToStep <=  ExecutionStep.Preprocessor

Reply:
Could you add a comment to explain the optimization ?